app/stac_api/urls.py

```diff
@@ -45,4 +47,8 @@
         AssetDetail.as_view(),
         name='asset-detail'
     ),
+    path(
+        "favicon.ico",
+        RedirectView.as_view(url=staticfiles_storage.url("favicon.ico")),
+    ),
```

Comment:
This should not be needed and the static file should be served by Whitenoise.

Reply:
I confirm this is not needed with Whitenoise.